front/src/pages/people/__stories__/People.add.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { expect } from '@storybook/jest';
+import type { Meta } from '@storybook/react';
+import { userEvent, within } from '@storybook/testing-library';
+
+import { AppPath } from '@/types/AppPath';
+import {
+  PageDecorator,
+  type PageDecoratorArgs,
+} from '~/testing/decorators/PageDecorator';
+import { graphqlMocks } from '~/testing/graphqlMocks';
+import { sleep } from '~/testing/sleep';
+
+import { People } from '../People';
+
+import { Story } from './People.stories';
+
+const meta: Meta<PageDecoratorArgs> = {
+  title: 'Pages/People/Add',
+  component: People,
+  decorators: [PageDecorator],
+  args: { routePath: AppPath.PeoplePage },
+  parameters: {
+    docs: { story: 'inline', iframeHeight: '500px' },
+    msw: graphqlMocks,
+  },
+};
+
+export default meta;
+
+export const AddNewPeople: Story = {
```

Comment:
```suggestion
export const AddNewPerson: Story = {
```

Reply:
AddNewPeople was the silliest name, I have ever given to a variable 🙃